lib/publiccloud/gcp_client.pm

```diff
@@ -32,7 +32,12 @@ sub init {
     $self->project_id($data->{project_id});
     $self->account($data->{client_id});
     assert_script_run('source ~/.bashrc');
-    (is_sle('>=15')) ? assert_script_run("chronyd -q 'pool time.google.com iburst'") : assert_script_run('ntpdate -s time.google.com');
+    if (is_sle('>=15')) {
+        script_run("killall chronyd");    # kill it in case it's running
```

Comment:
```suggestion
        # kill chronyd in case it's running
        script_run("killall chronyd && sleep 5 && if pgrep chronyd; then killall -9 chronyd; fi");
```

Suggestion for being .... thorough.